### ISSUE TITLE
Reversing a scale does not cause the marginal plots to reverse

### ISSUE THREAD

**Issue by daattali** [daattali/ggExtra#94]
Could be related to #81 et al

```
p <- ggplot(mtcars, aes(x = wt, y = drat)) +
    geom_point()

ggMarginal(p)
ggMarginal(p + scale_x_reverse())
```

![image](https://user-images.githubusercontent.com/952340/36286852-8bfb9774-127f-11e8-8d75-413172d0de91.png)

![image](https://user-images.githubusercontent.com/952340/36286842-845236f4-127f-11e8-870d-076a903c40ba.png)

**Comment by crew102** [comment]
Yep, related to #81. I just put together a fix for #81 that only included a
fix for limits issues, and was actually wondering if something like this
occurs as well. It's happening b/c the scale from the scat plot isn't
getting applied to the marginals. I think I have a more general fix that
will fix this and #81

**Comment by daattali** [comment]
fixed by @crew102
